lib/gui/app.js

```diff
@@ -214,6 +215,24 @@ app.run(($timeout) => {
   })
 
   driveScanner.on('error', (error) => {
+    if (error.message === 'LIBUSB_ERROR_ACCESS') {
+      return permissions.isElevated().then((elevated) => {
+        if (!elevated) {
+          // Stop all adapters but notably usbboot, remove usbboot, and start the remaining adapters
+          driveScanner.stop()
+          driveScanner.adapters.delete('usbboot')
```

Comment:
You should be able to use `driveScanner.unsubscribe('usbboot')` instead of `driveScanner.adapters.delete('usbboot')` here

Reply:
Does he also need to stop and start the driveScanner before unsubscribing, or is that all handled internally?

Reply:
Done this now, and also fixed a small bug in `.unsubscribe()`.

Reply:
@lurch yeah, at this time you still need to stop the scanner before unsubscribing – which doesn't make sense – I'll fix that in a new PR.